docs/design/definition-schema.md

```diff
@@ -0,0 +1,51 @@
+# Image Definition Schema
+
+## Schema Fields
+
+TBD
+
+## Versioning
+
+There are three types of changes that may be made to the definition schema between releases:
+* New optional additions
+* New required additions that would break existing definitions
+* Changes to existing fields (renaming, changing types)
```

Comment:
Nit: Removing a field is also a possibility.

Reply:
Good call, I'll add it.